style.css

```diff
@@ -400,3 +400,214 @@ form input,
   background-color: rgba(255, 255, 255, 0.3);
   border: rgba(255, 255, 255, 1);
 }
+
+@media screen and (min-width: 992px) {
+  #navbar {
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding: 20px 0;
+    background-color: #fff;
+  }
+
+  .links {
+    display: block;
+    background: none;
+    backdrop-filter: blur(0);
+    padding: 0;
+    mix-blend-mode: normal;
+    overflow: visible;
+    width: 100%;
+    height: 0;
+  }
+
+  #hamBtn {
+    display: none;
+  }
+
+  #hamBtnClose {
+    display: none;
+  }
+
+  .links li {
+    margin: 0 30px;
+  }
+
+  #toolbar {
+    margin-left: 155px;
+  }
+
+  .nav__links {
+    margin-top: 20px;
+    padding-right: 155px;
+    display: flex;
+    justify-content: flex-end;
+  }
+
+  .nav__links a {
+    color: #344563;
+    font-size: 15px;
+    font-weight: 500;
+  }
+
+  .background {
+    background-image: url(imgs/mobile-background.svg);
+    background-color: #fff;
+    background-repeat: no-repeat;
+    background-size: 90%;
+    height: 100%;
+    border-radius: 0 0 0 110px;
+  }
+
+  .headline {
+    display: flex;
+    flex-direction: column;
+    padding: 150px 260px 0 260px;
+  }
+
+  .headline h1 {
+    font-size: 48px;
+    line-height: 60px;
+  }
+
+  .headline p {
+    padding-right: 350px;
+  }
+
+  .hl-paragraph {
+    padding-right: 110px;
+    font-size: 20px;
+    font-weight: 300;
+  }
+
+  .hl-button {
+    width: fit-content;
+  }
+
+  .work-1 {
+    margin-top: 100px;
+  }
+
+  #portfoliosection {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .card-work {
+    display: flex;
+    width: 1170px;
+    height: 445px;
+  }
```

Comment:
- [ ] Following the project requirements, you should stick to the Figma design as much as you can. In the template that you have chosen, the position of the image in the card is different in each card. But in your design, you implemented the images at the same position in all the cards. See the comparison below 👇:
- Your design:
<img width="939" alt="image" src="https://user-images.githubusercontent.com/77190203/155863351-f5fc0d95-3f5f-43eb-9727-63f3a58bb42f.png">

- Figma design:
![image](https://user-images.githubusercontent.com/77190203/155863395-0f2869c2-4564-4be0-852c-a82837189286.png)

Reply:
I solved that thing in the last commit!

Reply:
- I started reviewing your project before you pushed your last commit.